docs/whatisk8s.md

```diff
@@ -0,0 +1,51 @@
+# What is Kubernetes?
+
+The Kubernetes project was started by Google in 2014. 
```

Comment:
I moved this down in a subsequent PR. It logically belongs with the builds-on-our-experience sentence. And it's really awkward to have it first.